NOTICE

```diff
@@ -0,0 +1,21 @@
+Agoric SDK
+Copyright 2018-2022 Agoric Systems Operating Company.
+
+This product includes software developed at
```

Comment:
Does this need to mention our license?  or is that in the LICENSE file?

Reply:
That is the recommendation from Apache on how to apply their license, and born out by examples sited on the ticket.